apis/python/src/tiledb/vector_search/module.py

```diff
@@ -36,15 +36,15 @@ def load_as_matrix(
     a = tiledb.ArraySchema.load(path, ctx=tiledb.Ctx(config))
     dtype = a.attr(0).dtype
     if dtype == np.float32:
-        m = tdbColMajorMatrix_f32(ctx, path, 0, 0, 0, size, 0, timestamp)
+        m = tdbColMajorMatrix_f32(ctx, path, 0, 0, 0, size, 0, timestamp, True)
```

Comment:
I don't love this interface to `tdbMatrix`.  I think what I would rather see (and I think have done elsewhere for something else) is to use an overload so that if we want to read the whole thing, we don't pass in a limit at all.  

(Using special values of variables can be bad if the special value is actually a valid value.)

Reply:
Sure thanks, updated the constructors so that callers can use the appropriate one without this flag.